.github/workflows/run_mdk.yml

```diff
@@ -0,0 +1,53 @@
+name: run_mdk
```

Comment:
Same as for the other pipeline: can you add a few comment lines at the top of this pipeline to summarize what it does, its dependencies and checks/products/artifacts it produces?